project_stock/models/project_task.py

```diff
@@ -121,17 +121,25 @@ def _compute_stock_state(self):
                         task.stock_state = state
                         break
 
-    @api.depends("move_ids", "move_ids.quantity_done")
+    @api.depends("move_ids", "move_ids.quantity")
     def _compute_unreserve_visible(self):
+        price_dp = self.env["decimal.precision"].precision_get(
+            "Product Unit of Measure"
+        )
         for item in self:
             already_reserved = item.mapped("move_ids.move_line_ids")
-            any_quantity_done = any([m.quantity_done > 0 for m in item.move_ids])
-            item.unreserve_visible = not any_quantity_done and already_reserved
+            any_quantity = any(
+                [
+                    not float_is_zero(m.quantity, precision_digits=price_dp)
```

Comment:
Are we allowing negative quantities?

Reply:
I hadn't thought about negative amounts, actually. 
The quantity done (`quantity`) cannot be defined, so IMO it is not a "real" possibility.

Reply:
Well, I think the previous `> 0` is enough then.